### ISSUE TITLE
user can't tell if they are logged in with new onboard.js

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-next#342]
testing `next.giveth.io`

with the new onboard.js the wallet address/user ID changes instantly on the giveth UI when a user changes their wallet in metamask.. however even though the UI shows the change the user is NOT in fact logged in.. 

![image.png](https://images.zenhubusercontent.com/5fff1c86d883b3d9a858ddbc/76313539-3e18-4917-b226-1a7bfb97f54b)

this means that a user might mistakenly be making donations while they are not logged in, this causes problems because that also means they will not receive any emails and their interactions will not be logged into our analytics. 

the other problem is the flow for a user to log-in is quite cumbersome and not at all obvious
upon arrival if a user wants to log in and visit projects they must:
click their user portrait(top right) -> click my accoutnt -> click sign -> click Giveth -> click projects ... 

ideally the user should be able to log in while never having to leave the page they are on.. 

we should have a way of showing the user from the navbar they are not logged in and having a button to press that pops-up the metamask sign to log-in

**Comment by mohammadranjbarz** [comment]
@divine-comedian 
It's on staging and ready for test

**Comment by divine-comedian** [comment]
testing `next.giveth.io` 

going through the flow I arrive on giveth.io, metamask shows my user portrait in the top right, without signing any message I go to donate to a project, when I click the `DONATE` button it first asks me to sign a message then metamask proceeds to the transaction window to send my donation. 

no issues encountered in this process! looks good!

**Comment by RamRamez** [comment]
@divine-comedian Thanks a lot. Did you also test segments analytics data?
The data should be ok for registered users, and for unregistered users should be anonymous.

**Comment by divine-comedian** [comment]
> @divine-comedian Thanks a lot. Did you also test segments analytics data? The data should be ok for registered users, and for unregistered users should be anonymous.

yes! I did check on segment and the information is displayed correctly as if a "logged in" user made a donation.
